src/renderer/src/lib/components/Input.svelte

```diff
@@ -1,15 +1,29 @@
 <script lang="ts">
+	import { createEventDispatcher } from 'svelte';
+
 	import type { InputProps } from '$types';
 
+	const dispatch = createEventDispatcher();
+
 	export let props: InputProps;
 	export let value: string | null = null;
+	export let autocomplete: string | null = null;
 	export let files: FileList | null = null;
 
+	function handleKeydown(event: KeyboardEvent) {
+		dispatch('keydown', event);
+	}
+
 	$: ({ type, class: iptClass = 'input placeholder-white text-sm mb-2', ...rest } = props);
 </script>
 
 {#if type === 'file'}
 	<input type="file" bind:files {...rest} class={iptClass} />
 {:else}
-	<input bind:value {...props} class={iptClass} />
+	<input on:keydown={handleKeydown} bind:value {...props} class={iptClass} />
+	{#if autocomplete && value}
+		<span style="left: {43 + value.length * 7}px" class="absolute top-[9px] opacity-50"
```

Comment:
I think this span needs a `pointer-events: none` CSS attribute so that when one clicks on it it selects the input element below.